README.md

```diff
@@ -27,6 +27,14 @@ The location at which the `jenkins-cli.jar` jarfile will be kept. This is used f
 
 Jenkins plugins to be installed automatically during provisioning. You can always install more plugins via the Jenkins UI at a later time, but this is helpful in getting things up and running more quickly.
 
+    jenkins_connection_delay: 5
+
+The amount of time to wait between connection attempts to the Jenkins web interface to verify it has started.
+
+    jenkins_connection_retries: 12
```

Comment:
Let's go back to 300 seconds (so `60` for the default here), just to be generous. The first Jenkins startup can take a while.